contracts/feature-tests/use-module/tests/gov_module_whitebox_test.rs

```diff
@@ -0,0 +1,603 @@
+const THIRD_USER_ADDRESS_EXPR: &str = "address:third-user";
+
+pub struct Payment {
+    pub token: Vec<u8>,
+    pub nonce: u64,
+    pub amount: u64,
+}
+
+fn world() -> ScenarioWorld {
+    let mut blockchain = ScenarioWorld::new();
+    blockchain.set_current_dir_from_workspace("contracts/features-tests/use-module");
+
+    blockchain.register_contract(USE_MODULE_PATH_EXPR, use_module::ContractBuilder);
+    blockchain
+}
+
+fn setup() -> ScenarioWorld {
+    let mut world = world();
+
+    world.set_state_step(
+        SetStateStep::new()
+            .put_account(
+                OWNER_ADDRESS_EXPR,
+                Account::new()
+                    .nonce(1)
+                    .esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(INITIAL_GOV_TOKEN_BALANCE)),
+            )
+            .new_address(OWNER_ADDRESS_EXPR, 1, USE_MODULE_ADDRESS_EXPR)
+            .put_account(
+                FIRST_USER_ADDRESS_EXPR,
+                Account::new()
+                    .nonce(1)
+                    .esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(INITIAL_GOV_TOKEN_BALANCE)),
+            )
+            .put_account(
+                SECOND_USER_ADDRESS_EXPR,
+                Account::new()
+                    .nonce(1)
+                    .esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(INITIAL_GOV_TOKEN_BALANCE)),
+            )
+            .put_account(
+                THIRD_USER_ADDRESS_EXPR,
+                Account::new()
+                    .nonce(1)
+                    .esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(INITIAL_GOV_TOKEN_BALANCE)),
+            ),
+    );
+
+    // init
+    let use_module_whitebox =
+        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
+    let use_module_code = world.code_expression(USE_MODULE_PATH_EXPR);
+
+    world.whitebox_deploy(
+        &use_module_whitebox,
+        ScDeployStep::new()
+            .from(OWNER_ADDRESS_EXPR)
+            .code(use_module_code),
+        |sc| {
+            sc.init_governance_module(
+                managed_token_id!(GOV_TOKEN_ID),
+                managed_biguint!(QUORUM),
+                managed_biguint!(MIN_BALANCE_PROPOSAL),
+                VOTING_DELAY_BLOCKS,
+                VOTING_PERIOD_BLOCKS,
+                LOCKING_PERIOD_BLOCKS,
+            );
+        },
+    );
+
+    world.set_state_step(SetStateStep::new().block_nonce(10));
+
+    world
+}
+
+pub fn propose(
+    world: &mut ScenarioWorld,
+    proposer: &Address,
+    gov_token_amount: u64,
+    dest_address: &Address,
+    endpoint_name: &[u8],
+    args: Vec<Vec<u8>>,
+) -> usize {
+    let use_module_whitebox =
+        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
+
+    let mut proposal_id = 0;
+
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new().from(proposer).esdt_transfer(
+            GOV_TOKEN_ID,
+            0,
+            &rust_biguint!(gov_token_amount),
```

Comment:
I think it should work without the rust_biguint! we should have direct conversion from small int